ChangeLog

```diff
@@ -29,14 +34,14 @@
 
 	* R/Attributes.R: Correct how cppFunction() deals with multiple
 	depends arguments
-	
```

Comment:
nit: please avoid whitespace-only changes when possible; consider configuring your editor to only trim whitespace on the lines you've edited

Reply:
That hit me the other day too.  So I just googled it for The One True Editor: https://www.emacswiki.org/emacs/DeletingWhitespace#toc11   

Will give that a spin myself.

Reply:
Sorry for not noticing the auto formatting. I'll disable auto formatting for the project.